code-style/javascript/README.md

```diff
@@ -0,0 +1,64 @@
+JavaScript
+====
+
+[![JS Logo][producti]][product]
+
+We write a lot of JavaScript. As the Web continues to grow and as users expect more dynamic interaction with websites, JavaScript has taken on an increasingly large role. However, we believe that it's also important to know when _not_ to use JavaScript. We prefer semantic HTML and CSS solutions whenever possible, and only reach for JavaScript to enhance these capabilities. We find that this approach helps to ensure that our work is accessible to the largest number of people possible.
+
+Style
+-----
+
+Consistent code styling helps our teams work together better and can help prevent common JavaScript pitfalls. We rely heavily on manual code reviews to ensure the quality and functionality of the code we're writing, but prefer to automate code style enforcement as much as possible.
+We use [ESLint][eslint] with a [custom configuration][eslint_config] to help enforce consistent code style.
+
+Here are a few guidelines we like to follow:
+
+- Prefer [ES6][es6] syntax, including
+  - [import][import]
+  - [spread operator][spread]
+
+Modern Code, Old Browsers
+-------------------------
+
+Browser support is a constantly moving target. While we like to be forward-thinking with our code, it's also important to remain backwards-compatible. We use [Babel][babel] and Webpack to allow us to use the most modern and awesome features of JavaScript while supporting the widest range of browsers.
+
+Static Types
+------------
+
+We like the dynamic nature of JavaScript, but we also recognize the usefulness of static type checking. We like to use [TypeScript][typescript] when the need arises.
+
+Frameworks / Libraries
+------------------------
+
+JavaScript has a rich ecosystem of libraries. Here are some of the tools we like to use.
+
+- [React][react]
+- [Vue][vue]
+- [Redux][redux]
+
+Testing
+-------
+
+Automated testing is an important part of our development process. Here are some of the tools we use to test.
```

Comment:
Not sure if I should add some details about testing strategy, or should that be its own section of the standard?

Reply:
I think testing gets it's own section

Reply:
Cool. I think we can leave this for now and then link to the testing section from here once that's in place.